ddtrace/_monkey.py

```diff
@@ -178,6 +181,76 @@ class ModuleNotFoundException(PatchException):
     pass
 
 
+class IncompatibleModuleException(PatchException):
+    def __init__(self, message, installed_version=None):
+        super().__init__(message)
+        self.installed_version = installed_version
+
+
+def is_version_compatible(version, supported_versions_spec):
+    # type: (str, str) -> bool
```

Comment:
```suggestion
def is_version_compatible(version: str, supported_versions_spec: str) -> bool:
```

we don't need to use type comments anymore, we can just type the code.

Reply:
same elsewhere

Reply:
done.